w1lldone/dumask.id
# Feature/notifications starter kit

- Setup notifications
- Setup queue
- Send mail notification after report has been submitted

## Test the notifications
- Run `php artisan migrate`.
- Edit `.env`.
```
QUEUE_CONNECTION=database
```
- If you are using `php artisan serve` please restart it before continuing to the next step.
- Make sure you have one user having "operate stations" permission
- Submit a report
- Run queue worker `php artisan queue:listen`. The output should look like this
![image](https://user-images.githubusercontent.com/19965509/121118479-1aecf400-c844-11eb-8e1a-4ba544d55ac2.png)

- Check the email sent to your mailtrap account. The email should look like this
![image](https://user-images.githubusercontent.com/19965509/121118556-4079fd80-c844-11eb-952f-91dbbe1ab93e.png)
